site/source/docs/building_from_source/index.rst

```diff
@@ -11,23 +11,28 @@ Emscripten's core codebase, which is in the main "emscripten" repo, does not nee
 Building LLVM
 -------------
 
-For using the LLVM wasm backend (recommended), simply build normal upstream LLVM from the monorepo, including clang and wasm-ld (using something like ``-DLLVM_ENABLE_PROJECTS=lld;clang'``) and the wasm backend (which is included by default; just don't disable it), following `that project's instructions <http://llvm.org/docs/CMake.html>`_. For example, something like this can work:
+For using the LLVM wasm backend (recommended), simply build normal upstream LLVM from the `monorepo <https://github.com/llvm/llvm-project>`_, including clang and wasm-ld (using something like ``-DLLVM_ENABLE_PROJECTS='lld;clang'``) and the wasm backend (which is included by default; just don't disable it), following `that project's instructions <http://llvm.org/docs/CMake.html>`_. For example, something like this can work:
 
   ::
 
       mkdir build
-      cd build
-      cmake .. -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD="host;WebAssembly" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF
+      cd build/
+      cmake ../llvm -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD="host;WebAssembly" -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF
+      make -j$(nproc)  # no need to install
 
 Please refer to the upstream docs for more detail.
 
+The version to select for e.g. 1.38.43 is --- TODO ---.
+
 For using the older fastcomp backend, see :ref:`the fastcomp docs <building-fastcomp-from-source>`.
 
 Building Binaryen
 -----------------
 
 See the `Binaryen build instructions <https://github.com/WebAssembly/binaryen#building>`_.
 
+The version to select for e.g. 1.38.43 is --- TODO ---.
```

Comment:
Again, one would almost always want ToT.    We have version tags in binaryen to match the emscripten version, but normally if you want to make change you want to use ToT.